apps/src/code-studio/components/progress/StatusProgressDot.story.jsx

```diff
@@ -16,6 +16,27 @@ export default storybook => {
     }
   });
 
+  const defaultProps = {
+    viewAs: ViewType.Student,
+    courseOverviewPage: true,
+    postMilestoneDisabled: false,
+    signInState: SignInState.SignedIn,
+    saveAnswersBeforeNavigation: false,
+    level: {
+      icon: null,
+      ids: [5275],
+      kind: 'puzzle',
+      next: [2, 1],
+      position: 1,
+      previous: [7,15],
+      title: 1,
+      url: 'http://localhost-studio.code.org:3000/s/course1/stage/8/puzzle/1',
+    },
+    levelProgress: { 5275: TestResults.ALL_PASS },
+    lessonIsLockedForAllStudents: () => false
```

Comment:
lessonIsLockedForAllStudents was missing, but this seems like a good time to more clearly show what was changing between different stories (by using these default props).